services/theme_update.py

```diff
@@ -36,8 +38,8 @@ def get_headers(self, length):
 class MigratedUpdate(ThemeUpdate):
 
     def get_data(self):
-        if hasattr(self, 'row'):
-            return self.row
+        if hasattr(self, 'data'):
+            return self.data
```

Comment:
! So before, we were re-executing that query ...

Reply:
yep...